### PR TITLE
Add chart-operator into e2e test

### PR DESCRIPTION
**Pull request by tomahawk28**
Toward https://github.com/giantswarm/app-operator/pull/120

it would support chart-operator `app` CR into e2e test.

### CODE REVIEW

File: pkg/chartvalues/apiextensions_app_e2e_template.go [hunk 1]
```diff
@@ -1,42 +1,55 @@
...
+      secret:
+        name: "{{ .App.Config.Secret.Name }}"
+        namespace: "{{ .App.Config.Secret.Namespace }}"
+    kubeConfig:
+      inCluster: {{ .App.KubeConfig.InCluster }}
+      secret:
+        name: "{{ .App.KubeConfig.Secret.Name }}"
+        namespace: "{{ .App.KubeConfig.Secret.Namespace }}"
+    version: "{{ .App.Version }}"
+  - name: "chart-operator"
```
Comment (rossf7) [inline pkg/chartvalues/apiextensions_app_e2e_template.go:22]: This makes sense as its the app CR for bootstrapping chart-operator. The other app CR is configurable.

But can we have a comment here explaining?

**Review by rossf7 — APPROVED** [review]
LGTM once the chart-operator comment is there.

File: pkg/chartvalues/apiextensions_app_e2e_template.go (hunk 1 above)
Comment (tomahawk28) [reply pkg/chartvalues/apiextensions_app_e2e_template.go:22]: Sure, that could be easier for someone to read this later.

Comment (rossf7) [reply pkg/chartvalues/apiextensions_app_e2e_template.go:22]: Damn too late but can we make it `Added app CR for bootstrapping chart-operator`?

As the repo is e2etemplates so this is all for e2e tests. 😄

**Pull request merged**